lib/core/mixin.ts

```diff
@@ -27,15 +28,16 @@ export function mixinCore
   {
     authStateManager: AuthStateManager<M, S, O>;
     serviceManager: ServiceManager<M, S, O>;
+    tokenSet: TokenSet;
     
     constructor(...args: any[]) {
       super(...args);
 
       // AuthStateManager
       this.authStateManager = new AuthStateManager<M, S, O>(this);
 
-      // ServiceManager
-      this.serviceManager = new ServiceManager<M, S, O>(this, this.options.services);
+      this.tokenSet = new TokenSet(this);
```

Comment:
Something like:

```javascript
this.map = new Map();
const defaultTokenSet = new TokenSet(this);
this.map.set(DEFAULT_TOKEN_SET_KEY, defaultTokenSet);
```

Reply:
so authjs itself becomes the `manager` of different token sets, and TokenSet becomes a logic group for `TokenManager` and `ServiceManager`?

Reply:
personally, I think we should avoid put too much management logic in the top level instance, maybe rename `TokenSet` as `TokenSetManager`?